presto-main/src/main/java/io/prestosql/version/EmbedVersion.java

```diff
@@ -47,18 +46,23 @@
     private final MethodHandle callableConstructor;
 
     @Inject
-    public EmbedVersion(ServerConfig serverConfig)
+    public EmbedVersion(NodeVersion version)
     {
-        Class<?> generatedClass = createClass(serverConfig);
+        this(version.getVersion());
+    }
+
+    public EmbedVersion(String version)
+    {
+        Class<?> generatedClass = createClass(format("Presto_%s_", version));
```

Comment:
```suggestion
        Class<?> generatedClass = createClass(format("Presto_%s___", version));
```

-- that was intentional, to visually separate version string (which may be short or longer) from the timestamp digits unconditionally appended to the class name later on.

Reply:
There are two underscores with this change (the class code adds one). We added three here to separate when there were two versions (now there is only one). I’m fine making it longer, but it didn’t seem necessary.

Reply:
I updated this to match your suggestion.